mindsdb_native/libs/phases/model_interface/lightwood_backend.py

```diff
@@ -410,7 +424,7 @@ def train(self):
                     lightwood_config['mixer']['kwargs']['callback_on_iter'] = self.callback_on_iter
                     lightwood_config['mixer']['kwargs']['eval_every_x_epochs'] = eval_every_x_epochs / len(final_mixer_classes)
 
-                lightwood_config['mixer']['kwargs']['stop_training_after_seconds'] = stop_training_after_per_mixer
+                lightwood_config['mixer']['kwargs']['stop_training_after_seconds'] = training_time_per_mixer
```

Comment:
Maybe we can re-allocate any remaining time, for whenever a particular mixer is much faster than its pre-assigned time budget?

Reply:
Hmh, good idea, but honestly I'm to tired to implement right now and there's a bunch of ?s regrading the impl (i.e. which mixer gets the time budget). If you feel it's worth it please make an issue and one of us can get to it later.